2/src/styles/_companies.scss

```diff
@@ -0,0 +1,84 @@
+.companies {
+  padding-top: 50px;
+  position: relative;
+  margin-bottom: 140px;
+
+  &-item {
+    justify-content: space-around;
+    display: flex !important;
+  }
+
+  .text-wrapper {
+    max-width: 570px;
+    font-size: 24px;
+    color: $mainColor;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+
+    q {
+      display: block;
+    }
+
+    p {
```

Comment:
what if inside section `text-wrapper` will be `p` with different styles?

Reply:
added classes